src/main/java/org/prebid/server/bidder/edge226/Edge226Bidder.java

```diff
@@ -0,0 +1,134 @@
+import com.iab.openrtb.response.BidResponse;
+import io.vertx.core.http.HttpMethod;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.prebid.server.bidder.Bidder;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderCall;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.exception.PreBidException;
+import org.prebid.server.json.DecodeException;
+import org.prebid.server.json.JacksonMapper;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+import org.prebid.server.proto.openrtb.ext.request.edge226.Edge226ImpExt;
+import org.prebid.server.proto.openrtb.ext.response.BidType;
+import org.prebid.server.util.HttpUtil;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+public class Edge226Bidder implements Bidder<BidRequest> {
+
+    private static final TypeReference<ExtPrebid<?, Edge226ImpExt>> EXT_TYPE_REFERENCE = new TypeReference<>() {
+    };
+    private static final String PUBLISHER_IMP_EXT_TYPE = "publisher";
+    private static final String NETWORK_IMP_EXT_TYPE = "network";
+
+    private final String endpointUrl;
+    private final JacksonMapper mapper;
+
+    public Edge226Bidder(String endpointUrl, JacksonMapper mapper) {
+        this.endpointUrl = HttpUtil.validateUrl(Objects.requireNonNull(endpointUrl));
+        this.mapper = Objects.requireNonNull(mapper);
+    }
+
+    @Override
+    public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest request) {
+        final List<HttpRequest<BidRequest>> httpRequests = new ArrayList<>();
+
+        for (Imp imp : request.getImp()) {
+            try {
+                final Edge226ImpExt impExt = parseImpExt(imp);
+                final BidRequest modifiedBidRequest = makeRequest(request, imp, impExt);
+                httpRequests.add(makeHttpRequest(modifiedBidRequest, imp.getId()));
+            } catch (PreBidException e) {
+                return Result.withError(BidderError.badInput(e.getMessage()));
+            }
+        }
+
+        return Result.withValues(httpRequests);
+    }
+
+    private Edge226ImpExt parseImpExt(Imp imp) throws PreBidException {
+        try {
+            return mapper.mapper().convertValue(imp.getExt(), EXT_TYPE_REFERENCE).getBidder();
+        } catch (IllegalArgumentException e) {
+            throw new PreBidException("Invalid ext. Imp.Id: " + imp.getId());
+        }
+    }
+
+    private BidRequest makeRequest(BidRequest request, Imp imp, Edge226ImpExt impExt) {
+        final boolean hasPlacement = StringUtils.isNotBlank(impExt.getPlacementId());
+        final boolean hasEndpoint = !hasPlacement && StringUtils.isNotBlank(impExt.getEndpointId());
+        final Edge226ImpExt modifiedImpExt = impExt.toBuilder()
```

Comment:
NitPick. It will be easier to compare with go, if instead of toBuilder will be used `Edge226ImpExt.builder()`

Reply:
Then you can remove `(toBuilder = true)` part from model